src/common/reducers/repositories.js

```diff
@@ -2,10 +2,11 @@ import * as ActionTypes from '../actions/repositories';
 import { parseGitHubRepoUrl } from '../helpers/github-url';
 
 export function repositories(state = {
-  isFetching: false,
+  // has no server side content, so initial state is always 'fetching'
+  isFetching: true,
```

Comment:
Why is that?
It shouldn't be `isFetching` until action is called to start fetching the data.